specification/storagecache/resource-manager/Microsoft.StorageCache/stable/2021-05-01/storagecache.json

```diff
@@ -1576,7 +1576,11 @@
                 "description": "The client ID of the user-assigned identity."
               }
             }
-          }
+          },
```

Comment:
This implies create-only of just the userAssignedIdentities block. Does this really belong at a higher level (CacheIdentity) because all cache identity fields are create-only?

Reply:
Making the appropriate change.